features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/Comments/CommentOptions.jsx

```diff
@@ -146,10 +144,13 @@ class CommentOptions extends React.Component {
         const {
             classes, comment, editIndex, index, theme,
         } = this.props;
+        const username = (AuthManager.getUser() && AuthManager.getUser().name);
+        const canDelete = (comment.createdBy === username)
+            || (username === theme.custom.adminRole);
```

Comment:
Same as above, Shall we use scope name , or we could add a helper method to [`User` class](https://github.com/wso2/carbon-apimgt/blob/master/features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/data/User.js) as `isAdmin`.
Where we check for the `apim:admin` scope and return true if we found it in the scopes list

Reply:
@malinthaprasan 
WDYT ?

Reply:
Fixed